src/pages/reports/shippingReport.js

```diff
@@ -7,8 +7,8 @@ import { startReport } from "../reportsQuery.js";
 export const bptlShipReportsScreen = async (auth, route) => {
     const user = auth.currentUser;
     if (!user) return;
-    const username = user.displayName || user.email;
-    appState.setState({ username });
+    const username = user.displayName || user.email;;
+    appState.setState({ username, reportPage: null, reportPageBoxData: null, numReportPages: null});
```

Comment:
With growing data in state manager, it might be better to group data  based on functionality so that they can be separated well. For example, you may consider put the 3 pieces of data related to reports under a holder like this: `appState.setState({ username, reportData: {reportPage: null, reportPageBoxData: null, numReportPages: null}})`

Reply:
I was on the fence about this because it makes the state management more verbose -- all properties of the reportData object need to be handled with each state update. Agreed on code organization...updated. Thanks.